engine/src/test/java/com/arcadedb/query/sql/executor/AlterTypeExecutionTest.java

```diff
@@ -108,8 +108,21 @@ public void sqlAlterTypeName() {
     database.command("sql", "CREATE VERTEX TYPE Mpv");
     database.command("sql", "ALTER TYPE Mpv NAME Sedan");
     Assertions.assertNotNull(database.getSchema().getType("Sedan"));
+
     // Assertions.assertNull fails, hence the use of database.command()
     ResultSet result = database.command("sql", "SELECT FROM schema:types");
     Assertions.assertFalse(result.stream().anyMatch(x -> x.getProperty("name").equals("Mpv")));
+
+    database.command("sql", "CREATE PROPERTY Sedan.engine_number string");
+    database.command("sql", "CREATE INDEX ON Sedan(engine_number) UNIQUE");
+    database.begin();
+    // insert a random record
+    database.getSchema().getType("Sedan").newRecord().set("engine_number", "123").save();
```

Comment:
Thanks for the change. I meant creating an index on the old type and see what happens when you rename it. Is the index still working or does it need to be renamed too?

Reply:
will do!